.github/workflows/pr.yml

```diff
@@ -53,12 +59,23 @@ jobs:
             tlatools/org.lamport.tlatools/target/surefire-reports/TEST-*.xml
             tlatools/org.lamport.tlatools/target/surefire-reports/TESTS-TestSuites.xml
             tlatools/org.lamport.tlatools/target/surefire-reports/junit-noframes.html
+            tlatools/org.lamport.tlatools/target/code-coverage.exec
+            tlatools/org.lamport.tlatools/target/code-coverage-report/report.xml
     - name: Publish unit test results
       uses: mikepenz/action-junit-report@v4
       if: always()
       with:
           check_name: JUnit Test Report on ${{ matrix.operating-system }}
           report_paths: 'tlatools/org.lamport.tlatools/target/surefire-reports/TEST-*.xml'
+    - name: Publish code coverage results
```

Comment:
So the difference between uploading and publishing is that uploading makes them available as artifacts, but publishing makes the github actions bot comment on your PR?

Reply:
The upload will make the .exec file with the coverage available to download in the test assets page inside the related zip file (see the bottom of this page: https://github.com/FedericoPonzi/tlaplus/actions/runs/9800330483?pr=1).

The "publish" step will invoke a github action to publish a comment with the code coverage (see comment: https://github.com/FedericoPonzi/tlaplus/pull/1#issuecomment-2209602384). I've added more info to the issue: https://github.com/tlaplus/tlaplus/issues/981#issuecomment-2209598609 and inputs to the discussion are welcome!